main.py

```diff
@@ -0,0 +1,52 @@
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import col, split, row_number, explode
+from pyspark.sql import Window
+
+if __name__ == '__main__':
+    spark = SparkSession \
+        .builder \
+        .master("local[12]") \
```

Comment:
you can put `*` here to use all available logical cores 
and pls choose one style: `double` or `single` quotes